pkg/kn/commands/broker/create.go

```diff
@@ -71,5 +74,6 @@ func NewBrokerCreateCommand(p *commands.KnParams) *cobra.Command {
 		},
 	}
 	commands.AddNamespaceFlags(cmd.Flags(), false)
+	cmd.Flags().StringVarP(&className, "class", "c", "", "Broker class name.")
```

Comment:
I would add some example to the description (and ommit the 'name').

Also we should add the `--class` to the overall examples sections of this command.

Not sure about the `-c` shortcut, as I don't know if this is needed (i don't expect people to call the option often). So I propose to commit this.

Reply:
Should I mention the default broker class name (MTChannelBasedBroker) in case none is specified using the option?

Reply:
```suggestion
	cmd.Flags().StringVarP(&className, "class", "c", "", "Broker class like \'MTChannelBasedBroker\' or \'KafkaBroker\' (if available)")
```

something like this.